modules/system/opposed-test.js

```diff
@@ -161,8 +161,8 @@ export default class OpposedTest {
 
       opposeResult.other = opposeResult.other.concat(opposeResult.modifiers.message);
 
-      let attackerSL = parseInt(attackerTest.result.SL);
-      let defenderSL = parseInt(defenderTest.result.SL);
+      let attackerSL = parseInt(attackerTest.result.SL ?? 0);
```

Comment:
> when opposed test was changed to unopposed, sometimes NaN value was showing up for me.

Was this the fix? What do you mean by opposed to unopposed? Are there reproduction steps?

Reply:
Ok, so now i have problems to reproduce this as well, but the issue which i've been facing was related to result.SL being undefined and parseInt was returning me NaN.